# gVCA : Current vacuum value on Channel A.
#        The vacuum is provided in 1/1000 of relative vacuum.
#        Please note that this differs from the setpoint given in percent,
#        as extra accuracy is desirable on the actual vacuum.

uint16 gVCA

# gVCB : Same as the one of channel A.

uint16 gVCB
